compiler/luci/pass/src/QuantizeWeights.cpp

```diff
@@ -407,8 +407,11 @@ void QuantizeWeights::quantize_weights(luci::CircleConst *weights)
     {
       sym_wquant_per_channel(weights, scaling_factor, channel_dim_index);
     }
-    quantparam->min.clear();
-    quantparam->max.clear();
+    if (not save_min_max)
+    {
+      quantparam->min.clear();
+      quantparam->max.clear();
+    }
```

Comment:
I guess that this code can be removed? Because they will be removed in QuantizeWithMinMax if save_min_max is false. 
```suggestion
```

Reply:
I think yes, it can be good idea, thank you!